src/main/java/org/javacord/bot/commands/WikiCommand.java

```diff
@@ -18,15 +18,19 @@
 import java.util.Arrays;
 import java.util.List;
 import java.util.function.Predicate;
+import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 import java.util.stream.Collectors;
 
+import static java.lang.Math.min;
```

Comment:
Either remove the static import (checkstyle violation) or create a suppression/exception for java.lang.Math. Either is fine.